core/base/planarGraphLayout/MergeTreeVisualization.h

```diff
@@ -0,0 +1,930 @@
+  }
+
+  bool isConflictingBoundsX(std::tuple<float, float, float, float> first,
+                            std::tuple<float, float, float, float> second) {
+    return isConflictingBoundsXOneWay(first, second)
+           or isConflictingBoundsXOneWay(second, first);
+  }
+
+  bool
+    isConflictingBoundsYOneWay(std::tuple<float, float, float, float> first,
+                               std::tuple<float, float, float, float> second) {
+    double eps = std::numeric_limits<float>::epsilon();
+    return (std::get<2>(first) <= std::get<2>(second)
+            and std::get<2>(second) <= std::get<3>(first))
+           or (std::get<2>(first) <= std::get<3>(second)
+               and std::get<3>(second) <= std::get<3>(first))
+           or (isEqual<float>(std::get<2>(first), std::get<2>(second), eps)
+               or isEqual<float>(std::get<2>(second), std::get<3>(first), eps)
+               or isEqual<float>(std::get<2>(first), std::get<3>(second), eps)
+               or isEqual<float>(std::get<3>(second), std::get<3>(first), eps));
+  }
+
+  bool isConflictingBoundsY(std::tuple<float, float, float, float> first,
+                            std::tuple<float, float, float, float> second) {
+    return isConflictingBoundsYOneWay(first, second)
+           or isConflictingBoundsYOneWay(second, first);
+  }
+
+  bool isConflictingBounds(std::tuple<float, float, float, float> first,
+                           std::tuple<float, float, float, float> second) {
+    return isConflictingBoundsX(first, second)
+           and isConflictingBoundsY(first, second);
+  }
+
+  bool
+    isConflictingBranchAndBound(std::tuple<float, float, float, float> first,
+                                std::tuple<float, float, float, float> second,
+                                FTMTree_MT *tree,
+                                ftm::idNode branchNodeOrigin,
+                                std::vector<float> &retVec,
+                                std::vector<LongSimplexId> &treeSimplexId) {
+    float xBranchNodeOrigin = retVec[treeSimplexId[branchNodeOrigin] * 2];
+    float xBranchNode
+      = retVec[treeSimplexId[tree->getNode(branchNodeOrigin)->getOrigin()] * 2];
+    float myMin = std::min(xBranchNode, xBranchNodeOrigin);
+    float myMax = std::max(xBranchNode, xBranchNodeOrigin);
+    auto branchBounds = std::make_tuple(myMin, myMax, 0, 0);
+    return isConflictingBoundsX(first, branchBounds)
+           and isConflictingBoundsY(first, second);
+  }
+
+  std::tuple<float, float, float, float>
+    shiftBranchBoundsTuple(std::tuple<float, float, float, float> branchBound,
+                           float realShift) {
+    return std::make_tuple(std::get<0>(branchBound) + realShift,
+                           std::get<1>(branchBound) + realShift,
+                           std::get<2>(branchBound), std::get<3>(branchBound));
+  }
+
+  void shiftBranchBounds(
+    std::vector<float> &retVec,
```

Comment:
is this the clang format indentation ?

Reply:
Yes it is indeed the clang format indentation.